server.js

```diff
@@ -0,0 +1,17 @@
+require("dotenv").config();
+const express = require("express");
+
+const server = express();
+
+server.use((req, res, next) => {
+  console.log(req.method, req.path);
+  next();
+});
```

Comment:
This is useless for the app so far. Please remove useless code.

Reply:
done